### PR TITLE
chore: use default port const

### PR DESCRIPTION
**Pull request by benschac**
remove hard code, use const

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1** <a href="https://app.graphite.dev/github/pr/natew/vxrn/1" target="_blank">Graphite</a> 
    * **PR #2** <a href="https://app.graphite.dev/github/pr/natew/vxrn/2" target="_blank">Graphite</a> 
      * **PR #3** <a href="https://app.graphite.dev/github/pr/natew/vxrn/3" target="_blank">Graphite</a> 
        * **PR #4** <a href="https://app.graphite.dev/github/pr/natew/vxrn/4" target="_blank">Graphite</a> 
          * **PR #5** <a href="https://app.graphite.dev/github/pr/natew/vxrn/5" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/natew/vxrn/5?utm_source=stack-comment).

File: packages/vite-native-client/src/client.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { DEFAULT_PORT } from './../../vxrn/src/utils/constants';
```
Comment (benschac) [inline packages/vite-native-client/src/client.ts:1]: @natew - should I use this one, add a consts package or just create a consts on `vite-native-client`

Comment (benschac) [reply packages/vite-native-client/src/client.ts:1]: makes sense why build would break

File: packages/vite-native-client/src/client.ts [hunk 2]
```diff
@@ -130,8 +130,8 @@ function warnFailedFetch(err: Error, path: string | string[]) {
   console.error(`${err}`)
   console.error(
     `[hmr] Failed to reload ${path}. ` +
-      `This could be due to syntax errors or importing non-existent ` +
-      `modules. (see errors above)`
+    `This could be due to syntax errors or importing non-existent ` +
```
Comment (benschac) [inline packages/vite-native-client/src/client.ts:133]: this is all default save file prettier

**Comment by natew** [comment]
i made port configurable i think its outdated but the pr for some reason is weird, i think i merged in the wrong order or something

**Pull request closed without merging**